server/app.js

```diff
@@ -190,7 +190,7 @@ app.get('/user/:username', async (req, res, next)=>{
 			});
 
 		for (const brew of brews) {
-			const match = googleBrews.findIndex((b)=>b.editId === brew.editId);
+			const match = googleBrews?.findIndex((b)=>b.editId === brew.editId) ?? -1;

```

Comment:
Doesn't `findIndex()` already return `-1` if it doesn't find a match?

If it's a case of handing when `googleBrews = empty`, how about we instead just add a check to exit the loop early if `googleBrews` is empty? Probably more readable and if there are lots of brews we might save some time.

Also, can we safely just pre-filter this loop so the only brews we loop over have `brew.googleId`, i.e., we already know they are stubs?